sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -94,13 +94,11 @@ void log_status(const char *prefix, status_t status) {
   int32_t arg;
   const char *message;
   char mod_id[4];
-  if (status_ok(status)) {
-    LOG_INFO("%skOk(%u)", prefix, status.value);
-  } else if (status_extract(status, &message, &arg, mod_id)) {
+  if (status_extract(status, &message, &arg, mod_id)) {
```

Comment:
could you make of the non-standard status format specifier we have here: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/runtime/print.c;drc=260c35e72003f1bd2f2a4bdf4e7d5606bf08b12e;l=42

so you don't have to extract everything manually here?